src/creatwth.cpp

```diff
@@ -118,13 +123,148 @@ static HMODULE WINAPI EnumerateModulesInProcess(HANDLE hProcess,
             continue;
         }
 
-        if (LoadNtHeaderFromProcess(hProcess, (HMODULE)pbLast, pNtHeader)) {
+        PVOID remoteHeader
+            = LoadNtHeaderFromProcess(hProcess, (HMODULE)pbLast, pNtHeader);
+        if (remoteHeader) {
+            if (pRemoteNtHeader) {
+                *pRemoteNtHeader = remoteHeader;
+            }
+
             return (HMODULE)pbLast;
         }
     }
     return NULL;
 }
 
+//////////////////////////////////////////////////////////////////////////////
+//
+// Find payloads in target process.
+//
+
+static PVOID FindDetourSectionInRemoteModule(HANDLE hProcess,
+                                             HMODULE hModule,
+                                             PIMAGE_NT_HEADERS32 pNtHeader,
```

Comment:
Can you add SAL annotations to the new functions please? 
https://docs.microsoft.com/en-us/cpp/code-quality/annotating-function-parameters-and-return-values?view=vs-2019